CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 GovReady-Q Release Notes
 ========================
 
-v999 (February XX, 2021)
+v0.9.1.52 (February 16, 2021)
```

Comment:
For branches going into develop we leave this section of the changelog alone. This is changed in the release branch off develop. Same comment for the VERSION file.

Reply:
I merged develop into my branch to get the up to date code before pushing.  Is there any mechanism to prevent this from happening?

Reply:
@alexanderward When we edit a new feature off from develop, we add CHANGELOG details underneath without touching the actual version number, like:

```markdown
v999 (Month XX, 2021)
---------------------------

[Draft additions to CHANGELOG go here]

v0.9.1.52 (February 16, 2021)
-----------------------------
```

This leads to some minor merge conflicts among feature branches and `develop` for the `CHANGELOG.md` file, but they are generally very easy to resolve.

Once all feature branches are merged into `develop` prior to a release candidate, we update `VERSION` and add version number and date to `CHANGELOG.md`.